setup.cfg

```diff
@@ -144,6 +144,8 @@ filterwarnings =
     ignore:Unknown config option:pytest.PytestConfigWarning
     ignore:matplotlibrc text\.usetex:UserWarning:matplotlib
     ignore:The 'text' argument to find\(\)-type methods is deprecated:DeprecationWarning
+    # https://github.com/astropy/astropy/issues/14881
+    ignore:.*product.*is deprecated as of NumPy 1\.25\.0.*:DeprecationWarning
```

Comment:
Would be nice to specify `h5py` here (or in the comment), but that's not worth rerunning the tests!

Reply:
I added a comment. I will let the CI run anyway in case someone else that does not have admin power wants to merge before I get back online again. Thanks for the review!